### PR TITLE
Follow up on vulnarability reports and upgrade all deps

### PR DESCRIPTION
**Pull request by stefanhoth**
Upgraded all available direct dependencies and added resolutions to force-upgrade for these downstream deps:

- minimist https://github.com/advisories/GHSA-vh95-rmgr-6w4m
- acorn https://github.com/advisories/GHSA-6chw-6frg-f759
- kind-of https://github.com/advisories/GHSA-6c8f-qphg-qjgp

Tests seemed fine but happy for @danez or @Rendez to raise concerns.

### CODE REVIEW

**Comment by Rendez** [comment]
Not sure about the resolutions, are they needed?

**Comment by danez** [comment]
Agree, also wouldn't do any resolutions. the security warnings are for dev only anyway.

**Comment by stefanhoth** [comment]
Thanks for the review. I've removed the resolutions (was to ensure the reported CVEs get addressed but I'm fine to remove them if they're dev only.

Also added two more package updates.

File: package.json [hunk 1]
```diff
@@ -45,32 +45,32 @@
...
+    "typescript": "^3.8.3"
   },
   "dependencies": {
     "@types/highlight.js": "^9.12.3",
-    "classnames": "^2.2.3",
-    "highlight.js": "^9.0.0",
-    "prop-types": "^15.5.6"
+    "classnames": "^2.2.6",
+    "highlight.js": "^10.0.1",
+    "prop-types": "^15.7.2"
```
Comment (danez) [inline package.json:73]: We shouldn't update all dependencies in package.json unless we know that one of them is the source of a security vul. Having bigger ranges here helps npm/yarn to deduplicate versions easily.

**Comment by stefanhoth** [comment]
@danez I'm glad this is tackled now.

**Pull request closed without merging**